backend/blueprint/api/utils/response.py

```diff
@@ -12,7 +12,7 @@ def __init__(self, statusCode):
     def generateMessage(self, message):
         self.id = uuid.uuid4()
         self.attributes = message
-        self.type = 'csvResponse'
+        self.type = 'imgResponse'
```

Comment:
why not making general method? why you are changing `type` according to PR sent?